src/__fixtures__/index.js

```diff
@@ -1,18 +1,17 @@
+/* eslint-disable import/no-commonjs */
 // @flow strict
 
-import { join } from 'path';
```

Comment:
Please add `FIXME` saying that you can't convert it to TS until the build system is supports converting ts files.
If you do workarounds please always add `FIXME` with an explanation.